modules/mono/glue/Managed/Files/Color.cs

```diff
@@ -379,7 +537,155 @@ public string ToHtml(bool include_alpha = true)
+        public static readonly Color Peru = new Color(0.80f, 0.52f, 0.25f);
+        public static readonly Color Pink = new Color(1.00f, 0.75f, 0.80f);
+        public static readonly Color Plum = new Color(0.87f, 0.63f, 0.87f);
+        public static readonly Color PowderBlue = new Color(0.69f, 0.88f, 0.90f);
+        public static readonly Color Purple = new Color(0.63f, 0.13f, 0.94f);
+        public static readonly Color RebeccaPurple = new Color(0.40f, 0.20f, 0.60f);
+        public static readonly Color Red = new Color(1.00f, 0.00f, 0.00f);
+        public static readonly Color RosyBrown = new Color(0.74f, 0.56f, 0.56f);
+        public static readonly Color RoyalBlue = new Color(0.25f, 0.41f, 0.88f);
+        public static readonly Color SaddleBrown = new Color(0.55f, 0.27f, 0.07f);
+        public static readonly Color Salmon = new Color(0.98f, 0.50f, 0.45f);
+        public static readonly Color SandyBrown = new Color(0.96f, 0.64f, 0.38f);
+        public static readonly Color SeaGreen = new Color(0.18f, 0.55f, 0.34f);
+        public static readonly Color SeaShell = new Color(1.00f, 0.96f, 0.93f);
+        public static readonly Color Sienna = new Color(0.63f, 0.32f, 0.18f);
+        public static readonly Color Silver = new Color(0.75f, 0.75f, 0.75f);
+        public static readonly Color SkyBlue = new Color(0.53f, 0.81f, 0.92f);
+        public static readonly Color SlateBlue = new Color(0.42f, 0.35f, 0.80f);
+        public static readonly Color SlateGray = new Color(0.44f, 0.50f, 0.56f);
+        public static readonly Color Snow = new Color(1.00f, 0.98f, 0.98f);
+        public static readonly Color SpringGreen = new Color(0.00f, 1.00f, 0.50f);
+        public static readonly Color SteelBlue = new Color(0.27f, 0.51f, 0.71f);
+        public static readonly Color Tan = new Color(0.82f, 0.71f, 0.55f);
+        public static readonly Color Teal = new Color(0.00f, 0.50f, 0.50f);
+        public static readonly Color Thistle = new Color(0.85f, 0.75f, 0.85f);
+        public static readonly Color Tomato = new Color(1.00f, 0.39f, 0.28f);
+        public static readonly Color Turquoise = new Color(0.25f, 0.88f, 0.82f);
+        public static readonly Color Violet = new Color(0.93f, 0.51f, 0.93f);
+        public static readonly Color WebGreen = new Color(0.00f, 0.50f, 0.00f);
+        public static readonly Color WebGray = new Color(0.50f, 0.50f, 0.50f);
+        public static readonly Color WebMaroon = new Color(0.50f, 0.00f, 0.00f);
+        public static readonly Color WebPurple = new Color(0.50f, 0.00f, 0.50f);
+        public static readonly Color Wheat = new Color(0.96f, 0.87f, 0.70f);
+        public static readonly Color White = new Color(1.00f, 1.00f, 1.00f);
+        public static readonly Color WhiteSmoke = new Color(0.96f, 0.96f, 0.96f);
+        public static readonly Color Yellow = new Color(1.00f, 1.00f, 0.00f);
+        public static readonly Color YellowGreen = new Color(0.60f, 0.80f, 0.20f);
```

Comment:
My main concern with this is code completion tools will have tons of options and it would be difficult to look through to find what you want. To keep preset colors separate from other stuff, I'd suggest a new `Colors` static class which only stores preset `Color` objects (but this is something to discuss with @neikeq first).

Also, this doesn't match the preset Vectors etc (when I added the preset Vectors I followed the example of Quat.Identity).

Reply:
Your right! I had totally intended to have get only properties.  Will fix it up.

Reply:
OK, these have been changed  to properties.

I agree that a large volume infrequently constants can lower the signal to noise ratio.  Moving them out to a separate class sounds like a good solution. I am happy enough to move them to once a consensus has been reached.